src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -140,7 +141,19 @@ trait BeamHelper extends LazyLogging {
             )
           ) ++ {
             if (parsedArgs.useCluster)
-              Map("beam.cluster.clusterType" -> parsedArgs.clusterType.get.toString)
+              Map(
+                "beam.cluster.clusterType" -> parsedArgs.clusterType.get.toString,
+                "akka.actor.provider"      -> "akka.cluster.ClusterActorRefProvider",
+                "akka.remote.artery.canonical.hostname" -> parsedArgs.nodeHost.getOrElse(
+                  throw new RuntimeException("`nodeHost` is not set")
+                ),
+                "akka.remote.artery.canonical.port" -> parsedArgs.nodePort
+                  .map { _.toInt }
+                  .getOrElse(throw new RuntimeException("`nodePort` is not set")),
+                "akka.cluster.seed-nodes" -> util.Arrays.asList(
```

Comment:
Thinking about this more in depth now - we need to handle multiple addresses. I don't believe this handles that currently. Again - probably something for the parser itself, though

Reply:
We leave is at is,but in the future, I think with Kubernetes we should not provide seed-nodes as args, but rather use discovery to get seed node.

Reply:
Agreed. You choose one or the other. Since we're moving to discovery, then no need to spend much time on this now anyway - especially since this is not long lived. Of course this would affect recovery if it can't find an appropriate node - but again not an issue for our current demands and future plans